api/src/cli/index.ts

```diff
@@ -20,11 +18,6 @@ import usersPasswd from './commands/users/passwd.js';
 export async function createCli(): Promise<Command> {
 	const program = new Command();
 
-	if ((await isInstalled()) === true) {
-		const extensionManager = getExtensionManager();
-		await extensionManager.initialize({ schedule: false, watch: false });
-	}
-
 	await emitter.emitInit('cli.before', { program });
```

Comment:
Do we not need to initialize the extensions for the `emitter.emitInit` below it to do anything? 🤔

Reply:
ha ehh yeah.. Although it's a bit of a catch-22 as we can't init the extension manager there as the database may or may not exist 🤓

Reply:
Yes, we do 😅